### PR TITLE
feat: Display cards

### LINKED ISSUE #16: Research how to display cards
The player gets the same number of cards as the round number, which could be up to 20 cards if the game is played with three players. 

Research how we can display the cards (Android view, etc.) and what functionality we could use if the number of cards cannot be displayed on the screen simultaneously (horizontal scroll, multiple layers, etc.). 

Possible options: LibGDX

## Definition of done

- [x] Provide research results in the ticket
- [x] Make a branch with an MVP for the solution that should be used

### PR DESCRIPTION
**Pull request by TannerGabriel**
## This PR

- Adds `recyclerview` for displaying multiple cards in a horizontal format
- Dynamically loads the cards using the suit and value
- Allows the cards to be clicked and selected

Note: The card assets in this PR are just used for demo purposes and will be replaced in #26 

Cards unselected:
image

Card selected:
image

Closes: #16

### CODE REVIEW

**Review by BenjaminHac-Edu — COMMENTED** [review]
I suggest organizing the cards by placing them in a separate sub-dictionary within the drawables.

**Comment by TannerGabriel** [comment]
> I suggest organizing the cards by placing them in a separate sub-dictionary within the drawables.

I agree, but this can probably be done with the real assets in #26. But I can also change it here if you like (but we would need to delete all the drawable resources later).

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -8,4 +8,6 @@
     <string name="noAccountYet"><u>No account yet?\nCreate one</u></string>
     <string name="loginFailed">Login failed</string>
     <string name="registerFailed">Registriation failed</string>
+    <string name="card_image_description">Wizard card</string>
```
Comment (FRALEXAlexander) [inline app/src/main/res/values/strings.xml:11]: Lets stay with camel case form string names

Comment (TannerGabriel) [reply app/src/main/res/values/strings.xml]: The IDE's default setting is snake case, which is a bit annoying, but I changed it to camel case anyway. In the future, we can maybe consider switching back.

**Review by FRALEXAlexander — APPROVED** [review]
LGTM

**Pull request merged**